data/translations/en/state-data.json

```diff
@@ -269,13 +299,16 @@
     "more_info_link": "https://www.elections.il.gov/Default.aspx",
     "more_info_link_translated": "true",
     "more_info_link_language_selection": "",
-    "ip_deadline": "Available up to and including on Election Day",
-    "online_deadline": "16 days before Election Day",
-    "mail_received_deadline": "Must be received 28 days before Election Day",
-    "mail_postmarked_deadline": "",
+    "online_deadline": "Tuesday, October 18, 2022",
+    "ip_deadline": "Tuesday, November 8, 2022",
+    "mail_postmarked_deadline": "Must be postmarked by Thursday, October 06, 2022",
```

Comment:
Changed from postmarked to received. Which should we use? We can't use both.

Reply:
It changed to postmarked, we will not use the old received by

Reply:
confirmed